index.js

```diff
@@ -32,28 +32,72 @@ const fastifyMail = async (fastify, opts) => {
 
   // mail decorator configurations:
   const mail = {
-    createMessage: async function (recipients, templates, context) {
+    // createMessage: async function (recipients, templates, context, opts = {}) {
+    createMessage: async function (message, templatePath, context) {
```

Comment:
should this have a similar signature as sendMail (would it make more sense if it did, i don't know)?

Reply:
We're already de-structuring {opts} in sendmail. I didn't think we'd need to do it again.  Also  - Isn't it more explicit to show the reader exactly what this function uses? Also I don't really mind :)

Reply:
Cool, let's go with what you have then :)